packages/databyss-services/sources/SourceNotFoundError.js

```diff
@@ -0,0 +1,15 @@
+export class SourceNotFoundError extends Error {
```

Comment:
Let's make this a more generic `ResourceNotFoundError` thrown by `request.js` (see comment in `actions`)